ci/playbooks/meta_content_provider/meta_content_provider.yml

```diff
@@ -47,7 +47,9 @@
     - name: Build openstack services container when gating repo exists
       when:
         - "'os-net-config' not in zuul_change_list"
-        - _gating_repo.stat.exists
+        # Note: cimfw_tcib_force_container_build var is used to run build_containers
+        # role in the meta content provider irrespective of gating repo.
+        - _gating_repo.stat.exists or cimfw_tcib_force_container_build | default('false')
```

Comment:
There is a typo `cimwf` vs `cifmw`. Additionally, it would be good to follow the same pattern, like in other `build_containers`. Something like: `cifmw_build_containers_force` or similar.

Reply:
Done!